src/Microsoft.Graph.Core/Requests/BaseClient.cs

```diff
@@ -70,7 +69,7 @@ IAuthenticationProvider authenticationProvider
         /// <param name="httpClient">The customized <see cref="HttpClient"/> to be used for making requests</param>
         public BaseClient(
             string baseUrl,
-            HttpClient httpClient):this(new HttpClientRequestAdapter(new AnonymousAuthenticationProvider(), httpClient: httpClient) { BaseUrl = baseUrl })
+            HttpClient httpClient):this(new BaseGraphRequestAdapter(new AnonymousAuthenticationProvider(), httpClient: httpClient) { BaseUrl = baseUrl })
```

Comment:
why are we defaulting to an anonymous authentication provider here instead of requiring the authentication provider? especially since we don't have an auth handler anymore.

Reply:
This constructor has a couple of jobs
- Allow simple creating of a client with no authentication provider in the event the user wishes to have custom auth in the http middleware
- Create clients for scenarios like large file upload where there is no need for auth. 

In the event one has an authProvider/tokencredential, they will use the other two constrcutors above.

Reply:
ok, thanks, we can always review the usefulness of the base client class later since there is no way for the generated graph service client to inherit from it today. resolving.